### PR TITLE
Add BMW and Mini chassis codes

### PR DESCRIPTION
**Pull request by bogdan-calapod**
Related to #79, added an `enum` class to work out matches between the chassis code and a model number (e.g. "MINI Cooper S", "7 Series", etc.)

For now it's pretty basic, but should be enough for just some simple UX customization. We could add more info down the road if we want to customize or change logic based on the exact model.

### CODE REVIEW

File: app/src/main/assets/carapplications/ChassisKodes.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    E34("BMW", "5 Series"),
+    E32("BMW", "7 Series"),
+    E31("BMW", "8 Series"),
+    E30("BMW", "3 Series"),
+    E28("BMW", "5 Series"),
+    E24("BMW", "6 Series"),
+    E23("BMW", "7 Series"),
+    E21("BMW", "3 Series"),
+    E12("BMW", "5 Series"),
+    E82E("BMW", "1 Series");
```
Comment (hufman) [inline app/src/main/assets/carapplications/ChassisKodes.kt:89]: I think this E82E code might be a typo, can you check?

File: app/src/main/assets/carapplications/ChassisKodes.kt [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    E71("BMW", "X6"),
+    E70("BMW", "X5"),
+    E53("BMW", "X5"),
+    E26("BMW", "M1"),
+    E86("BMW", "Z4 Coupe"),
+    E89("BMW", "Z4 Roadster"),
+    E85("BMW", "Z4 Roadster"),
+    E25("BMW", "Concept Turbo"),
+
+    E9("BMW",	"2 Series"),
```
Comment (hufman) [inline app/src/main/assets/carapplications/ChassisKodes.kt:34]: These rows have slightly different spacing hehe

Comment (bogdan-calapod) [reply app/src/main/assets/carapplications/ChassisKodes.kt:34]: An artefact of the gimmicks I did to convert between formats 😁

File: app/src/main/assets/carapplications/ChassisKodes.kt (hunk 1 above)
Comment (bogdan-calapod) [reply app/src/main/assets/carapplications/ChassisKodes.kt:89]: Seems to be the Active variant of the [E82](https://en.wikipedia.org/wiki/BMW_1_Series_(E87)#2-door_coupe_(E82)). I've seen it in a couple other chassis code lists as well 🤔

File: app/src/main/assets/carapplications/ChassisKodes.kt [hunk 3]
```diff
@@ -0,0 +1,102 @@
+enum class ChassisCode(val brand: String, val chassis: String) {
```
[the remark this answers is not in the document]
Comment (bogdan-calapod) [reply app/src/main/assets/carapplications/ChassisKodes.kt]: Agreed - the "K" was a typo 😁

**Pull request merged**